src/wallet/rpcdump.cpp

```diff
@@ -961,7 +963,8 @@ UniValue ProcessImport(CWallet * const pwallet, const UniValue& data, const int6
                 pwallet->SetAddressBook(vchAddress, label, "receive");
 
                 if (pwallet->HaveKey(vchAddress)) {
-                    return false;
+                    result.pushKV("success", UniValue(true));
```

Comment:
Should be `true`?

Reply:
Since when `success` is `false` returns the associated `error`, in this situation since there's no associated error, `success` it should be true. Just like calling n times the `importprivkey` RPC for a given private key: it does not throw any error.

Reply:
Right, but maybe that behaviour is wrong? Search for `throw JSONRPCError(RPC_WALLET_ERROR, "The wallet already contains the private key for this address or script");`

Reply:
BTW, all previous exit points are exceptions, this is the only early return.

Reply:
> Right, but maybe that behaviour is wrong? Search for throw JSONRPCError(RPC_WALLET_ERROR, "The wallet already contains the private key for this address or script");

Right, so we can keep it consistent with the rest of the `importmulti` logic by throwing that error.